config/courses.json

```
@@ -8,7 +8,11 @@
     },
     "deploy": true,
     "devtoolsEnabled": true,
-    "inProduction": true
+    "inProduction": true,
+    "name": "spanish-from-english",
+    "source": "English",
```

I would use something more verbose but more specific:

```suggestion
    "sourceLanguage": {
      "name": "English",
    },
```